Integrations/AWS-EC2/AWS-EC2.py

```diff
@@ -0,0 +1,2679 @@
+
+
+def start_instances_command(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+
+    response = client.start_instances(InstanceIds=parse_resource_ids(args.get('instanceIds')))
+    if response['ResponseMetadata']['HTTPStatusCode'] == 200:
+        demisto.results("The Instances were started")
+
+
+def stop_instances_command(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+
+    response = client.stop_instances(InstanceIds=parse_resource_ids(args.get('instanceIds')))
+    if response['ResponseMetadata']['HTTPStatusCode'] == 200:
+        demisto.results("The Instances were stopped")
+
+
+def terminate_instances_command(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+
+    response = client.terminate_instances(InstanceIds=parse_resource_ids(args.get('instanceIds')))
+    if response['ResponseMetadata']['HTTPStatusCode'] == 200:
+        demisto.results("The Instances were terminated")
+
+
+def create_volume_command(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+    obj = vars(client._client_config)
+    kwargs = {'AvailabilityZone': args.get('availabilityZone')}
+
+    if args.get('encrypted') is not None:
+        kwargs.update({'Encrypted': True if args.get('encrypted') == 'True' else False})
+    if args.get('iops') is not None:
+        kwargs.update({'Iops': int(args.get('iops'))})
+    if args.get('kmsKeyId') is not None:
+        kwargs.update({'KmsKeyId': args.get('kmsKeyId')})
+    if args.get('size') is not None:
+        kwargs.update({'Size': int(args.get('size'))})
+    if args.get('snapshotId') is not None:
+        kwargs.update({'SnapshotId': args.get('snapshotId')})
+    if args.get('volumeType') is not None:
+        kwargs.update({'VolumeType': args.get('volumeType')})
+    if args.get('kmsKeyId') is not None:
+        kwargs.update({'KmsKeyId': args.get('kmsKeyId')})
+    if args.get('tags') is not None:
+        kwargs.update({
+            'TagSpecifications': [{
+                'ResourceType': 'volume',
+                'Tags': parse_tag_field(args.get('tags'))}]
+        })
+
+    response = client.create_volume(**kwargs)
+
+    data = ({
+        'AvailabilityZone': response['AvailabilityZone'],
+        'CreateTime': datetime.strftime(response['CreateTime'], '%Y-%m-%dT%H:%M:%SZ'),
```

Comment:
this can throw value error. better calculate it before the append and in a try block.